src/components/SearchTracePage/index.js

```diff
@@ -18,29 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import _values from 'lodash/values';
```

Comment:
@saminzadeh I have mixed feelings about the name, myself, for the same reason. I went with `_values` because 
- `values` would be an odd name for it as it's a function not a collection of values
  - Come to think of it, though, `_values` has basically the same issue
- `_values` is reminiscent of `_.values` 

I noticed another [project](https://github.com/lodash/babel-plugin-lodash#example) using the same approach, but I'm definitely open to alternatives... Maybe `valuesFn`? (or `lodashValues`? Long names are a bother, though.)

Yes, importing by function (also called cherry-picking, I think) yields smaller builds. Consistency in the project, in this respect, is a WIP.